### ISSUE TITLE
Read the docs no longer gets updated

### ISSUE THREAD

**Issue by icemac** [zopefoundation/Zope#485]
https://zope.readthedocs.io/ no longer gets updated.

I think this is because GitHub Services – wich we are using to inform RTD about change are deprecated. The last built on RTD was more than one month ago.

Please either fix the GitHub integration on the RTD side or add me (`icemac`) to the Zope maintainers on RTD: https://readthedocs.org/projects/zope/

**Comment by jugmac00** [comment]
I would also appreciate a small documentation how the build process works.

**Comment by hannosch** [comment]
Hey, I've added icemac as an admin to the Zope readthedocs project. I've also done a manual build of the docs, which did succeed.

Afterwards I've followed the docs at https://docs.readthedocs.io/en/stable/webhooks.html and created an "incoming Github web hook" on the readthedocs side and created the matching outgoing webhook on the Github project.

I think this should the all that was required.

Cheers!
